pkgs/by-name/me/mednafen-server/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "mednafen-server";
+  version = "0.5.2";
+
+  src = fetchurl {
+    url = "https://mednafen.github.io/releases/files/mednafen-server-${finalAttrs.version}.tar.xz";
+    hash = "sha256-uJmxaMW+bydfAXq8XDOioMoBOLUsi5OT2Tpbbotsp3Y=";
+  };
+
+  strictDeps = true;
+
+  postInstall = ''
+    install -m 644 -Dt $out/share/mednafen-server standard.conf
+  '';
+
+  meta = {
```

Comment:
If you make `meta = with lib; {` then you can remove `lib.` from anything below this line.

Reply:
`with` does not work like the distributive axiom of a field (as suggested by your remark), and its rules are unintuitive.

https://nix.dev/anti-patterns/language#with-attrset-expression

https://github.com/NixOS/nix/issues/490

https://github.com/NixOS/nixpkgs/issues/208242

Reply:
That last one is about overuse of `with`. In this case, I wouldn't consider using `with lib;` for assigning values in `meta` to be an overuse.

Reply:
Until a nasty `with` scope invasion appears:


NixOS/nixpkgs#292468 (comment)